nilearn/decoding/decoder.py

```diff
@@ -1290,24 +1258,30 @@ def __sklearn_tags__(self):
         See the sklearn documentation for more details on tags
         https://scikit-learn.org/1.6/developers/develop.html#estimator-tags
         """
-        # TODO
-        # get rid of if block
-        # bumping sklearn_version > 1.5
-        # see https://github.com/scikit-learn/scikit-learn/pull/29677
-        tags = super().__sklearn_tags__()
-        if SKLEARN_LT_1_6:
-            return tags
+        return super().__sklearn_tags__()
```

Comment:
`_more_tags` can be defined in `_BaseDecoder` and deleted from all subclasses. If I am not mistaken it is the same in all inheriting classes.

Reply:
The definition of  `__sklearn_tags__` can be deleted.  If there is not a definition in current class, it will already call the super class method.

Reply:
pretty sure I have tried in some tests and was getting some tests failure: can you double check to see if we reach the same conclusion?

I am not too fussed about having some extra _more_tags() lying around as they will be deleted anyway in the future.